pandas/tests/io/sas/test_sas7bdat.py

```diff
@@ -216,6 +216,15 @@ def test_zero_variables(datapath):
         pd.read_sas(fname)
 
 
+def test_zero_rows(datapath):
+    # GH 18198
+    fname = datapath("io", "sas", "data", "zero_rows.sas7bdat")
+    df = pd.read_sas(fname)
+    tm.assert_frame_equal(
+        df, pd.DataFrame([{"char_field": "a", "num_field": 1.0}]).iloc[:0]
```

Comment:
Nit: Could you set these to `result` and `expected` and then call `tm.assert_frame_equal(result, expected)`?